proof/refine/AARCH64/Arch_R.thy

```diff
@@ -1234,49 +1370,56 @@ lemma arch_performInvocation_corres:
   apply (clarsimp simp: archinv_relation_def)
   apply (cases ai)
 
-  \<comment> \<open>InvokeVSpace\<close>
-  apply clarsimp
-  subgoal sorry (* FIXME AARCH64 TODO: handle InvokeVSpace *)
-
-     \<comment> \<open>InvokePageTable\<close>
+       \<comment> \<open>InvokeVSpace\<close>
+       apply (clarsimp simp: performARMMMUInvocation_def perform_vspace_invocation_def
+                             performVSpaceInvocation_def)
+       apply ((corres simp: perform_flush_def do_flush_def doFlush_def
+                     corres: corres_machine_op_Id_dc
+                     term_simp: vspace_invocation_map_def
+               | corres_cases_both simp: vspace_invocation_map_def)+)[1]
```

Comment:
I've been asking myself the same thing. I mainly stopped because finding a new name was too hard :-). Do you have a suggestion? (I'd do that in a separate PR)